src/batch-client/index.ts

```diff
@@ -499,6 +512,20 @@ export class ZimbraBatchClient {
 			namespace: Namespace.Account
 		}).then(res => mapValuesDeep(res._attrs, coerceStringToBoolean));
 
+	public recoverAccount = ({
+		channel = SetRecoveryAccountChannelType.email,
```

Comment:
channel is a mandatory param in schema.graphql, so don't think this default value will ever get used